### ISSUE TITLE
"Value cannot be null" error when importing with via a custom emulator profile with no file extensions configured

### ISSUE THREAD

**Issue by alextiley** [psychonic/Playnite-EmuLibrary#9]
An unhandled error is being throw during some imports. I did some debugging and it seems to occur whenever you've configured a custom emulator profile that does not contain at least one file extension. Whilst EmuLibrary probably can't import without an extension, it confused me for a little while - luckily I can read code but others not so fortunate 😄 It might make sense to either suppress the error and handle this more gracefully, or show a message detailing a misconfigured profile.

Happy to take a look and raise a PR if you have any opinions/steer. 👍 

**Playnite error notification:**

![image](https://github.com/psychonic/Playnite-EmuLibrary/assets/1131174/75fafc81-d197-4e13-94f7-675192315f17)

**Logs (playnite.log):**
```
11-01 20:10:21.288|INFO |MainViewModelBase:Importing games from EmuLibrary plugin.
11-01 20:10:21.295|ERROR|MainViewModelBase:Failed to import games from plugin: EmuLibrary
System.ArgumentNullException: Value cannot be null.
Parameter name: source
   at System.Linq.Enumerable.Where[TSource](IEnumerable`1 source, Func`2 predicate)
   at EmuLibrary.Settings.EmulatorMapping.get_ImageExtensionsLower()
   at EmuLibrary.RomTypes.SingleFile.SingleFileScanner.<GetGames>d__7.MoveNext()
   at EmuLibrary.EmuLibrary.<GetGames>d__26.MoveNext()
   at Playnite.Database.GameDatabase.ImportGames(LibraryPlugin library, CancellationToken cancelToken, PlaytimeImportMode playtimeImportMode) in C:\projects\playnite\source\Playnite\Database\GameDatabase.cs:line 1223
   at Playnite.ViewModels.MainViewModelBase.ImportLibraryGames(LibraryPlugin plugin, CancellationToken token) in C:\projects\playnite\source\Playnite\ViewModels\MainViewModelBase.cs:line 506
```

**Emulator configuration:**

![image](https://github.com/psychonic/Playnite-EmuLibrary/assets/1131174/26055430-c95c-4625-8032-51bb2df4356c)

**Comment by alextiley** [comment]
I believe the culprit for the unhandled exception is over here - https://github.com/psychonic/Playnite-EmuLibrary/blob/master/EmuLibrary/Settings/EmulatorMapping.cs#L128

**Comment by psychonic** [comment]
Thanks for the report. I have a plan to add more validation of mappings, tying into the Playnite Plugin SDK's support for settings verification to tie into the UI as gracefully as possible. This will be able to block saving on the settings page with a popup explaining why.

It will look something like this:
![image](https://github.com/psychonic/Playnite-EmuLibrary/assets/844118/8bc67b11-74e2-4af7-acdc-30eabdc9eb18)

**Comment by psychonic** [comment]
This should be fixed now in v3.4.1 via the aforementioned Settings validation.
